package.json

```diff
@@ -14,11 +14,12 @@
     "start": "webpack-dev-server",
     "test": "jest --passWithNoTests",
     "typescript": "tsc",
+    "update-translations": "tx pull",
     "watch-tests": "jest --watch"
   },
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged && concurrently 'yarn:lint' 'yarn:test' 'yarn:typescript'"
+      "pre-commit": "pretty-quick --staged && concurrently 'yarn:lint' 'yarn:test' 'yarn:typescript' 'yarn:autodoc' 'yarn:update-translations' && git add README.md translations/*.json"
```

Comment:
Wow, that's all it took?  I guess we currently have the Bamboo server set up to commit to the PIH repo